spec/parallel_tests/test/runner_spec.rb

```diff
@@ -108,7 +108,18 @@ def call(*args)
         ParallelTests::Test::Runner.should_receive(:runtimes).
           and_return({"aaa" => 0, "bbb" => 3, "ccc" => 1, "ddd" => 2})
         result = call(["aaa", "bbb", "ccc", "ddd", "eee"], 3, isolate: true, single_process: [/^aaa/], group_by: :runtime)
-        result.should == [["aaa"], ["bbb", "eee"], ["ccc", "ddd"]]
```

Comment:
the code does not use rand ... this should not be different on different machines :(

what ruby version are you running this with ? / maybe group_by or something like that are different ...

how about making them not be exactly the same -> force an order ?